app/components/Layout/NavBar/index.jsx

```
@@ -1,82 +1,79 @@
-          style={{ cursor: "pointer" }}
-        >
-          <img src={Logo.src} alt="Resonate Logo" className="logo-icon" />
-          <span className="logo-text">Resonate</span>
-        </div>
+				{/* Hamburger button for mobile */}
+				<button
+					className="hamburger"
+					onClick={() => setOpen(!open)}
+					aria-label="Toggle navigation"
+					aria-expanded={open}
+				>
```

Biome (2.4.7)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)